src/Lecture2.hs

```diff
@@ -150,14 +164,209 @@ You're free to define any helper functions.
        treasure besides gold (if you already haven't done this).
 -}
 
--- some help in the beginning ;)
+newtype Gold = Gold Int
+newtype XP = XP Int
+newtype Health = Health Int deriving (Eq, Ord)
```

Comment:
You can define newtypes as records. So you won't need to implement functions like `healthToInt` manually.
```suggestion
newtype Health = Health { unHealth :: Int } deriving (Eq, Ord)
```